src/simulation/dynamics/spinningBodies/spinningBodiesOneDOF/spinningBodyOneDOFStateEffector.cpp

```diff
@@ -70,13 +70,14 @@ void SpinningBodyOneDOFStateEffector::Reset(uint64_t CurrentClock)
     }
     // Ensure user specified valid angular limits
     if (this->theta_max < this->theta_min) {
-        bskLogger.bskLog(
-            BSK_ERROR, "theta_max (%f) must be greater than theta_min (%f).", this->theta_max, this->theta_min);
+        throw std::invalid_argument("theta_max (" + std::to_string(this->theta_max)
```

Comment:
I would like to keep the BSK_ERROR message as well, as this is expected behavior of a BSK module across the framework.  I don't mind if you also include the throw argument as well.